.env.example

```
@@ -0,0 +1,18 @@
+API_KEY=
+DATABASE_URL=
+
+# For production
+RACK_ENV=production
+RAILS_ENV=production
+RAILS_LOG_TO_STDOUT=enabled
+RAILS_SERVE_STATIC_FILES=enabled
+WEB_CONCURRENCY=2
+
+# For development
+RACK_ENV=development
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The RACK_ENV key is duplicated

(DuplicatedKey)